.npmignore

```diff
@@ -41,3 +41,7 @@ book.json
 .eslintrc
 .eslintignore
 .yarnrc
+babel.config.json
+.flowconfig
+Gemfile
+Gemfile.lock
```

Comment:
By removing these, you're further breaking the ability to `npm explore foo && npm install-test`, which doesn't seem worth the mere 1k/4k size decrease (the eslint and yarn config files should be included too, ofc)

Reply:
Never used `npm explore` and never heard that someone use it)

Reply:
It's why every npm package I solely maintain, or create, also publishes all its tests to npm.

Reply:
I'm with @ljharb here, not sure if the size improvement is worth this PR